packer_ubuntu1404.json

```diff
@@ -70,7 +70,8 @@
       "inline" : [
         "sudo apt-cache search build-essential",
         "sudo apt-get clean",
-        "sudo apt-get update"
+        "sudo apt-get update",
+        "sudo apt-get upgrade"
```

Comment:
Pretty sure you'll want a -y in there...

Reply:
Ya my bad. fixed it